patterns/terraform/main.tf

```diff
@@ -2,15 +2,14 @@ provider "aws" {
   region = local.region
 }
 
-# This provider is required for ECR to autheticate with public repos. Please note ECR authetication requires us-east-1 as region hence its hardcoded below.
-# If your region is same as us-east-1 then you can just use one aws provider
+# Required for public ECR where ACK artifacts are hosted
 provider "aws" {
-  alias  = "ecr"
   region = "us-east-1"
+  alias  = "virginia"
```

Comment:
Why changing from ecr to virginia?

Reply:
The alias ecr might give the impression that the provider is specifically for interacting with ECR, which is not the case. The provider is needed because ECR authentication requires the us-east-1 region. The new alias virginia more accurately represents the region rather than implying a direct interaction with ECR.

Reply:
> The alias ecr might give the impression that the provider is specifically for interacting with ECR

That was the intention

> The new alias virginia more accurately represents the region rather than implying a direct interaction with ECR

This is not true - this could lead folks to believe they can change the region. They cannot

Reply:
interesting. I saw it [here](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/705eb80cfacf563f74f760261c2154d2fd9f1d87/examples/karpenter/main.tf#L5C1-L8C2) and I thought it was intentional.

Reply:
same [here](https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/blob/c003cad83f930a16395a5e4cddd7daa1d74c76a3/tests/complete/main.tf#L5C1-L9C2).